.travis.yml

```diff
@@ -27,7 +27,6 @@ before_script:
   - sudo apt-get update
   # beanstalkd
   - sudo apt-get install -qq --force-yes beanstalkd
-  - sudo beanstalkd -d -l 127.0.0.1 -p 11300
```

Comment:
Why this change

Reply:
looks like the daemon is now started on apt-get install, port was reported as used in that line